config/manifest/eksa-components.yaml

```diff
@@ -3526,6 +3531,144 @@ status:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.1
+  creationTimestamp: null
+  name: tinkerbelltemplateconfigs.anywhere.eks.amazonaws.com
+spec:
+  group: anywhere.eks.amazonaws.com
+  names:
+    kind: TinkerbellTemplateConfig
+    listKind: TinkerbellTemplateConfigList
+    plural: tinkerbelltemplateconfigs
+    singular: tinkerbelltemplateconfig
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: TinkerbellTemplateConfig is the Schema for the TinkerbellTemplateConfigs
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: TinkerbellTemplateConfigSpec defines the desired state of
+              TinkerbellTemplateConfig
+            properties:
+              template:
+                description: Workflow represents a workflow to be executed.
```

Comment:
> Workflow represents ...

What do you mean here? This is a template.

Reply:
I think this got auto generated from the types comment. I just copied the upstream types in.
I can make this more descriptive 👍